userland/src/add_two_ints_client.cpp

```diff
@@ -37,8 +37,10 @@ int main(int argc, char ** argv)
 
   auto node = rclcpp::Node::make_shared("add_two_ints_client");
 
-  auto client = node->create_client<userland_msgs::AddTwoInts>("add_two_ints");
-  auto request = std::make_shared<userland_msgs::AddTwoInts::Request>();
+  auto client = node->create_client<
+userland_msgs::srv::AddTwoInts>("add_two_ints");
+  auto request = std::make_shared<
+userland_msgs::srv::AddTwoInts::Request>();
```

Comment:
[style] Why are these lines lacking any leading whitespace? I would expect them to the be indented once from the indention level of the line they started on. (similar cases through out the pr)